libs/community/langchain_community/chat_models/bedrock.py

```diff
@@ -308,7 +308,7 @@ def _combine_llm_outputs(self, llm_outputs: List[Optional[dict]]) -> dict:
         final_output = {}
         for output in llm_outputs:
             output = output or {}
-            usage = output.pop("usage", {})
+            usage = output.get("usage", {})
```

Comment:
I see that @baskaryan made some changes to` _combine_llm_outputs` to fix some lint and naming (thanks btw, looks much cleaner!).
I may be wrong, but instead of using `.get`, you may have to add `final_output["usage"] = final_usage` after line 314 to make sure that the dict accumulating the counters is saved back in the returned dict. Otherwise you simply return the tokens of the last iteration over llm_outputs.

Reply:
Thank you! However, I noticed that `final_output["usage"] = final_usage` is already present after line 314. I've also added a test for `_combine_llm_outputs`. Could you please take a look? I believe it functions correctly.

Reply:
You are right! The file viewer cut the code so I didn't see the full function :D Thanks for the test, it LGTM

Reply:
You are tight! The github file viewer cut the code short and I didn't see the full function :D . Thanks for the test, it lgtm